src/Components/SpotifyControl/SpotifyControl.js

```diff
@@ -4,6 +4,11 @@ import useSpotifyAuth from "Store/selectors/useSpotifyAuthData"
 import bragiIcon from "Assets/images/bragi-icon.png"
 import SpotifyControlView from "./SpotifyControlView";
 
+/**
+ * Component for current song information showing play/pause button, cover image, name of song and artists
+ * @returns {JSX.Element} SpotifyControlView with current song information
+ * @constructor
```

Comment:
`@constructor` is supposed to be used for functions that are classes. Our components are just normal functions, so it should be removed.

Reply:
Alright thanks, it was auto generated and wasn't sure what to do with it

Reply:
Ah, probably because the name starts with a capital, like classess usually